### ISSUE TITLE
Add support for multi-guilds

### ISSUE THREAD

**Issue by Kirollos** [Trickyyy/Oxide.Ext.Discord#5]
Extension currently supports handling only one guild.

**Comment by Kirollos** [comment]
[6d2287d] Main changes in the API so far that plugin developers need to know:

**DiscordClient:**
- `Guild DiscordServer` is replaced with `List<Guild> DiscordServers`
- Updated functions to the following:
```cs
public void RequestGuildMembers(string guild_id, string query = "", int limit = 0);
public void RequestGuildMembers(Guild guild, string query = "", int limit = 0);
public void UpdateVoiceState(string guildID, string channelId, bool selfDeaf, bool selfMute);
```
- Addition of the following:
```cs
public Guild GetGuild(string id); // Gets Guild object from the list
public void UpdateGuild(string g_id, Guild newguild); // Updates Guild objectref into the list
```

Misc:
- Helpers.DateCreation returns DateTime of the creation of a specific ID. It accepts ID of guild, user, channel, etc..

**Comment by Kirollos** [comment]
- DiscordClient . DiscordServer is added temporarily to default to the first detected guild during testing phase.
- Topic will be closed when 1.0.4 is fully released without issues related to multi-guilds
